servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -4091,7 +4122,7 @@ RendererSceneRenderRD::RendererSceneRenderRD(RendererStorageRD *p_storage) {
 
 	uint32_t textures_per_stage = RD::get_singleton()->limit_get(RD::LIMIT_MAX_TEXTURES_PER_SHADER_STAGE);
 
-	low_end = GLOBAL_GET("rendering/driver/rd_renderer/use_low_end_renderer");
+	low_end = is_low_end();
```

Comment:
feels like this may not be needed any more, make sure to check

Reply:
After discussing on chat we're going to attempt to remove this as the mobile renderer replaces the functionality but in a new PR.